### PR TITLE
[Chore] Remove firebase and users microservice

### PR DESCRIPTION
**Pull request by manekenpix**
## Description

Since we're going to use `supabase` as telescope's main database, there's no need for a `firebase` emulator in development.
This PR removes all references to the emulator and the `users` service.

## Steps to test the PR
- Run `docker-compose --env-file ./config/env.development up --build`
- Make sure `telescope` runs as expected

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — DISMISSED** [review]
I actually think we can remove the Users service in a follow-up too.  Supabase will be the Users service going forward.

The only thing we need to consider is what our "admin" story is.

**Review by DukeManh — DISMISSED** [review]

**Comment by DukeManh** [comment]
Oh, wait, removing this would break the users test because it uses the firebase emulator. @manekenpix, we should remove the users service in this PR as well.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -198,34 +198,6 @@ services:
       - 'traefik.http.middlewares.strip_feed_discovery_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.feed-discovery.middlewares=strip_feed_discovery_prefix'
 
-  # users service
```
Comment (humphd) [inline docker/docker-compose.yml:201]: Line 100 above, remove the `users` service from the `sso`'s depends:

```
Service 'sso' depends on service 'users' which is undefined.
```

**Review by humphd — CHANGES_REQUESTED** [review]
Looks good, one thing.

**Review by menghif — APPROVED** [review]

**Comment by humphd** [comment]
Rebased and fixed in https://github.com/Seneca-CDOT/telescope/commit/aca9f75e2c1359bfa02dfd38f4eb6947f1a05644.

**Pull request closed without merging**